packages/medusa/src/api/routes/admin/customer-groups/get-customer-group.ts

```diff
@@ -0,0 +1,49 @@
+import { CustomerGroupService } from "../../../../services"
+import { FindParams } from "../../../../types/common"
+import { validator } from "../../../../utils/validator"
+import { defaultAdminCustomerGroupsRelations } from "."
+
+/**
+ * @oas [get] /customer-group/{id}
+ * operationId: "GetCustomerGroups"
+ * summary: "Retrieve a CustomerGroup"
+ * description: "Retrieves a Customer Group."
+ * x-authenticated: true
+ * parameters:
+ *   - (path) id=* {string} The id of the Customer Group.
+ * tags:
+ *   - CustomerGroup
+ * responses:
+ *   200:
+ *     description: OK
+ *     content:
+ *       application/json:
+ *         schema:
+ *           properties:
+ *             customerGroup:
+ *               $ref: "#/components/schemas/customer-group"
+ */
+export default async (req, res) => {
+  const { id } = req.params
+
+  const validated = await validator(FindParams, req.query)
+
+  const customerGroupService: CustomerGroupService = req.scope.resolve(
+    "customerGroupService"
+  )
+
+  let expandFields: string[] = []
+  if (validated.expand) {
+    expandFields = validated.expand.split(",")
+  }
+
+  const findConfig = {
+    relations: expandFields.length
+      ? expandFields
+      : defaultAdminCustomerGroupsRelations,
+  }
+
+  const customerGroup = await customerGroupService.retrieve(id, findConfig)
+
+  res.json({ customerGroup })
+}
```

Comment:
In the client `@medusajs/medusa-js` we rely on each endpoint exporting a type that specifies which parameters the endpoint takes. The convention for how this type should be named is: 
- `[Admin | Store] + [operationId] + "Params"` for query parameters
- `[Admin | Store] + [operationId] + "Req"` for query request payload

This endpoint's `operationId` is `GetCustomerGroupsGroup` and as such we expect the query params to be defined by an exported type: `AdminGetCustomerGroupsGroupParams`.

In this case you can do:

```suggestion
}

export class AdminGetCustomerGroupsGroupParams extends FindParams {}
```

I understand if this might seem a bit weird, but the goal here is that in the future we will be able to write a code-gen script that would take care of automatically building the `@medusajs/medusa-js` client from these types.